packages/integrations/vercel/README.md

```diff
@@ -281,6 +281,26 @@ export default defineConfig({
 });
 ```
 
+### streaming
+
+**Type:** `boolean`<br>
+**Available for:** Serverless
+
+Use this property to set whether the serverless function will stream the response to the browser.
```

Comment:
```suggestion
Determines whether or not the serverless function will stream its response to the browser, sending components as they are rendered.

The default value is `false`. Set this value to `true` in your Vercel adapter configuration to enable streaming.
```
I'm assuming the default is `false`, if you have to set it to be true to take effect? I think it would be good to state that. Maybe something like above?

Also, to check:

1. In the caution just under here in docs: https://docs.astro.build/en/guides/server-side-rendering/#astrorequestmethod , and a little further down the page in the streaming section, it *sounds* like streaming is automatically used for SSR / `output: 'server'`. So at first I was confused to even see this PR! If this is not the case, then do the other docs need updating? Just trying to reconcile what's here vs on the existing SSR page.

2. That caution I linked to explains that because of streaming, you have to use features that modify the response headers only at the page level and not the component level (because the response headers may already be sent by the time some components render). Should a similar warning exist or be linked to from here re: enabling streaming? Is this a point you think might be important for someone to know, or potentially a breaking change in an existing project if they enable this?

Reply:
> I'm assuming the default is false

Effectively, yes, but it's not out default, it is Vercel's. So, I'm a bit hesitant to explicitly to document it on our site.

1. In this case it is the platform that interfered and made it all-at-once. I was not aware that Vercel did this, and neither was Nate. I just stumbled upon a suspicious Vercel reference doc and had to try it out to confirm. We have to explicitly tell Vercel to actually allow streaming.

2. No, the limitations apply the same no matter what the platform does once Astro is done.

Reply:
Also, the suggestion looks good to me but it does make it inconsistent with descriptions of the other config options in the README.

Reply:
OK, so thinking about this then! I'm looking back through earlier convo and noticing Nate thought it was true by default.

So, I agree, this actually does sound in keeping with what we expected and documented! No worries about those extra concerns! 

Give me a few minutes more to think about updated wording.